faq.md

```diff
@@ -113,6 +113,49 @@ testing.
 If you have an entire directory that you want to "exclude" from REUSE compliance
 testing, you can [use a DEP5 file](#bulk-license).
 
+## How do I exclude certain files from REUSE compliance testing? {#exclude-lines}
+
+In order to make the tool ignore a specific section containing strings that may
+falsely be detected as copyright or license statements, you can wrap it within
+the two comments `REUSE-IgnoreStart` and `REUSE-IgnoreEnd`.
+
+Please note that this MUST NOT be used to ignore valid copyright and licensing
+information by yourself or a third party. The ignore blocks MUST only be used
+for marking blocks that may trigger false-positive detections and errors.
```

Comment:
I would remove the RFC-2119 capitalised 'MUST' and 'MUST NOT' from the FAQ.

Reply:
Done